aptos-move/block-executor/src/executor.rs

```diff
@@ -518,26 +513,24 @@ where
                         Ok(finalized_group) => {
                             // finalize_group already applies the deletions.
                             if finalized_group.is_empty() != metadata_is_deletion {
-                                return Err(Error::FallbackToSequential(resource_group_error(
-                                    format!(
+                                return Err(code_invariant_error(format!(
```

Comment:
Oh good point, didn't realize it was specialized to that.

Reply:
Yep - we had discussions about it - we lose a lot of error information

Reply:
changed the message now so it can be used for delayed fields and resource group errors, the msg will anyway clarify what was triggered. let's revisit later.